data/unused/unused_9800.asm

```diff
@@ -1,19 +1,17 @@
-macro_9800: MACRO
-x = 0
-rept \1
-y = 0
-rept $100 / \1
-	db (x + y) & $ff
-y = y + \1
-endr
-x = x + 1
-endr
+MACRO macro_9800
+	FOR X, \1
+		DEF y = 0
+		rept $100 / \1

```

Comment:
Oh sorry, one more `rept` that should be a `for`

Reply:
Okay; pushed this.